components/tidb_query_datatype/src/codec/data_type/bit_vec.rs

```diff
@@ -3,7 +3,7 @@
 /// A boolean vector, which consolidates 64 booleans into 1 u64 to save space.
 ///
 /// `BitVec` is mainly used to implement bitmap in ChunkedVec.
-#[derive(Debug, PartialEq, Clone)]
+#[derive(Debug, PartialEq, Clone, Eq, Ord, PartialOrd)]
```

Comment:
`BitVec` should not derive the default `Eq` and `Ord`. Two `BitVec` could contain the same data but with different capacity, causing issues in this case.

Reply:
After `set` migrate to use `usize` represent bitmap, we don't need to use `BitVec` anymore, so this change will be reverted.